agent/config/runtime_test.go

```diff
@@ -514,6 +514,17 @@ func TestBuilder_BuildAndValidate_ConfigFlagsAndEdgecases(t *testing.T) {
 				rt.DataDir = dataDir
 			},
 		},
+		{
+			desc: "-http-max-header-bytes",
+			args: []string{
+				`-http-max-header-bytes=1`,
+				`-data-dir=` + dataDir,
+			},
+			patch: func(rt *RuntimeConfig) {
+				rt.HTTPMaxHeaderBytes = 1
+				rt.DataDir = dataDir
+			},
+		},
```

Comment:
This test case can be removed now that we don't have a command line flag. I believe this is the only test failure, so once it is removed the tests should pass.

Reply:
👍 completely missed removing that.  Let me know if you see anything more @dnephin